packages/tuono-router/src/components/RouterContext.tsx

```diff
@@ -20,3 +36,88 @@ export function getRouterContext(): React.Context<Router> {
 
   return routerContext
 }
+
+interface RouterContextProviderProps {
+  router: Router
+  children: ReactNode
+  serverSideProps?: ServerRouterInfo
+}
```

Comment:
This should be moved right before `RouterContextProvider`. 
Props should be alongside component declarations.
```suggestion
```